src/js/extension-anchor.js

```diff
@@ -51,6 +55,42 @@ var AnchorExtension;
             return this.form;
         },
 
+        getTemplate: function () {
+
+            var template = [
+                '<input type="text" class="medium-editor-toolbar-input" placeHolder="', this.base.options.anchorInputPlaceholder, '">'
```

Comment:
hmm, i've always just kind of written it as camelCase ... never had an issue where a browser didn't figure it out, but can lowercase for consistency if desired.